lib/modules/datasource/gitlab-tags/readme.md

```diff
@@ -9,12 +9,13 @@ To specify where to find a self-hosted GitLab instance, specify `registryUrl`. A
 **Usage Example**
 
 A real-world example for this specific datasource would be maintaining package versions in a config file.
-This can be achieved by configuring a generic regex manager in `renovate.json` for files named `versions.ini`:
+This can be achieved by configuring a generic custom manager in `renovate.json` for files named `versions.ini`:
```

Comment:
```suggestion
This can be achieved by configuring a custom manager in `renovate.json` for files named `versions.ini`:
```

Please replace all instances of "generic custom manager" with "custom manager"